Physics dataclass (#4268)
ukaea/PROCESS: documentation/source/physics-models/fusion_reactions/plasma_reactions.md

### Set global physics variables | `set_physics_variables()`

> This method sets the required physics variables in the `physics_variables` and `physics` module. It updates the global physics variables and module variables with the current instance's fusion power densities and reaction rates.

Reviewer: ```suggestion
This method sets the required physics variables on the `physics` data structure with the current instance's fusion power densities and reaction rates.
```
